npm/pkg/transport/pb/transport.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+package pb;
+option go_package = "github.com/Azure/azure-container-networking/npm/pkg/transport/pb;pb";
+
+import "google/protobuf/struct.proto";
+
+// DataplaneEvents represents the Service RPC exposed by the gRPC server.
+service DataplaneEvents{
+	rpc Connect(DatapathPodMetadata) returns (stream Events);
+}
+
+// DatapathPodMetadata is the metadata for a datapath pod
+message DatapathPodMetadata {
+  string id = 1; // Daemonset Pod ID
+  string node = 2; // Node name
```

Comment:
Do we need a separate ID ? because we can use Node name as the ID.

Reply:
I think pod name would be good to track (for future use probably)?

Reply:
nit: maybe better to be very explicit about the identifiers? `pod_id` and `node_name` seem less ambiguous to me.

Reply:
sure I can do that

Reply:
Also, do we want to introduce an "api_version" field to help us maintain the structure in long run for upgrade cases ? Both for PodMetadata and also events

Reply:
<Nit> we normally use "daemonPod" instead of DataplanePod, helps us with consistent naming.

Reply:
Sticking to Pod and Node to keep it generic. I don't understand why the file is showing the older version 
We have `pod` and `node` as the two fields

Reply:
> nit: maybe better to be very explicit about the identifiers? `pod_id` and `node_name` seem less ambiguous to me.

Changed it to `pod` and `node` to keep it simple. The id suffix is redundant IMO. WDYT?

Reply:
Don't let my feedback on this be a blocker for prototyping purposes btw :), just sharing my 2 cents. Thing about contracts is to keep ambiguity to a minimum IMO. `pod` could be understood as the name of the pod, or the uid of the pod, 2 different things. Maybe documentation could help in the proto file itself, but when used in the generated code, it would also have a bit of ambiguity.

Reply:
Sure I can update that to `pod_name` & `node_name`

Reply:
Done